.golangci.yml

```diff
@@ -56,12 +55,10 @@ linters:
     - gocyclo
     - godot
     - godox
-    - goerr113
     - gofmt
     - gofumpt
     - goheader
     - goimports
-    - gomnd
     - gomoddirectives
```

Comment:
These were renamed from `goerr113` --> `err113` and `gomnd` --> `mnd`. We should keep them in this list but with their new names rather than remove them entirely

Reply:
ahh good to know those new ones, thanks